### PR TITLE
Revert aws auth 2.4.6

### PR DESCRIPTION
**Pull request by leoluz**
Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [ ] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

File: hack/installers/checksums/awscli-linux-arm64-v2.4.6.zip.sha256 [hunk 1]
```diff
@@ -0,0 +1 @@
+095f5895976155ec570ea550ad81f92b88f1d55f1d7fc8813b30f329a82a83ba  awscli-linux-arm64-v2.4.6.zip
```
Comment (34fathombelow) [inline hack/installers/checksums/awscli-linux-arm64-v2.4.6.zip.sha256:1]: We would also need s390x & ppc64le binaries.   Last time I checked Amazon did not provide these.   That is one of the reasons we implemented the argocd-K8s-auth while reducing the size of the of the final image.

Comment (crenshaw-dev) [reply hack/installers/checksums/awscli-linux-arm64-v2.4.6.zip.sha256:1]: @34fathombelow we’re just using this branch to build a one-off image to narrow down an issue. Might be one of many experiments, ‘cause we’re very short on ideas. 😛

Comment (leoluz) [reply hack/installers/checksums/awscli-linux-arm64-v2.4.6.zip.sha256:1]: I see.. tks for pointing out. At this point, this branch is mainly an exercise to help us investigating a possible issue with `argocd-K8s-auth`. We are getting an elevated number of `Unauthorized` errors in our internal servers.

**Comment by leoluz** [comment]
Conclusion: While the new aws-auth binary could take advantage of caching tokens, it wasn't identified as the main reason why we are seeing an increase number of `Unauthorized` errors in ArgoCD 2.4.x.
I implemented a retry mechanism to address the issue and is available in the current patch 2.4.11.

**Pull request closed without merging**